bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -543,24 +629,22 @@ private void create_24_Tour(final StringBuilder sb) {
                 */
                if (isWeather) {
 
-                  if (UI.IS_SCRAMBLE_DATA) {
-                     tourWeather = UI.scrambleText(tourWeather);
-                  }
-
-                  if (isDescription) {
-                     // write spacer
-                     sb.append("<div>&nbsp;</div>");//$NON-NLS-1$
-                  }
-
-                  sb.append("<div class='title'>" + Messages.tour_editor_section_weather + "</div>" + NL); //$NON-NLS-1$ //$NON-NLS-2$
-                  sb.append("<p class='description'>" + WEB.convertHTML_LineBreaks(tourWeather) + "</p>" + NL); //$NON-NLS-1$ //$NON-NLS-2$
+                  sb.append(buildWeatherSection(tourWeather, isDescription));
                }
+
+               /*
+                * Tags
+                */
+               final Set<TourTag> tourTags = _tourData.getTourTags();
+               sb.append(buildTagsSection(tourTags, isDescription || isWeather));
             }
             sb.append("</div>" + NL); //$NON-NLS-1$
          }
          sb.append("</div>" + NL); //$NON-NLS-1$
 
-      } else {
+      } else
+
+      {
```

Comment:
Why this extra spacing ?

Reply:
Good catch.
Not sure why, maybe Eclipse ?